Formula/asc.rb

```diff
@@ -0,0 +1,22 @@
+class Asc < Formula
+  desc "App Store Connect API access via Swift command-line tool"
+  homepage "https://github.com/Blackjacx/Assist"
+  url "https://github.com/Blackjacx/Assist.git", tag: "0.1.1", revision: "6f07c2e97eb72bf2250b5bca888604007951c081"
```

Comment:
```suggestion
  url "https://github.com/Blackjacx/Assist.git",
      tag:      "0.1.1",
      revision: "6f07c2e97eb72bf2250b5bca888604007951c081"
```
Can't we use a tarball?

Reply:
What is the advantage?

Reply:
Git clones are slow and bad for caching.